src/ProjectTemplates/scripts/Run-Razor-Locally.ps1

```diff
@@ -1,9 +1,57 @@
-#!/usr/bin/env powershell
+#!/usr/bin/env pwsh
 #requires -version 4
 
+# This script packages, installs and creates a template to help with rapid iteration in the templating area.
 [CmdletBinding(PositionalBinding = $false)]
-param()
+param(
+    [Parameter(Mandatory = $false, Position = 0)]
+    [ValidateSet("net9.0", "net10.0")]
+    [string] $Framework = "net9.0",
+    [Parameter(Mandatory = $false)]
+    [switch] $ExcludeLaunchSettings,
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("None", "Individual")]
+    [string] $Auth = "None",
+    [Parameter(Mandatory = $false)]
+    [switch] $UseLocalDb,
+    [Parameter(Mandatory = $false)]
+    [switch] $NoHttps,
+    [Parameter(Mandatory = $false)]
+    [switch] $UseProgramMain,
```

Comment:
For future reference, I asked copilot to generate this, as well as the code that maps it into the variable by giving the output from `dotnet new <<template>> -h`. In case someone wants to update other scripts in the future.